masterlist.yaml

```
@@ -16038,7 +16038,7 @@ plugins:
         subs: [ '[Wyrmstooth](https://www.nexusmods.com/skyrimspecialedition/mods/45565/)' ]
         condition: 'not version("Wyrmstooth.esp", "1.19.2", >=)'
     clean:
-      - crc: 0x09BD8451
+      - crc: 0x1EC9136A
         util: 'SSEEdit v4.0.3'
       - crc: 0x8A598B0F
         util: 'SSEEdit v4.0.3'
```

0x09BD8451 is for the German version.
You should be using xEdit v4.0.3h (Hotfix 1), as it fixes some issue with cleaning.

```suggestion
      - crc: 0x1EC9136A
        util: 'SSEEdit v4.0.3h (Hotfix 1)'
      - crc: 0x09BD8451
        util: 'SSEEdit v4.0.3'
```